### PR TITLE
[IMP] (map_values): Support set & notset selectors

### PR DESCRIPTION
**Pull request by kurkop**
`True` as `old value` represents "is set", whereas `False` represets "is not set"

### CODE REVIEW

File: openupgradelib/openupgrade.py [hunk 1]
```diff
@@ -734,20 +735,35 @@ def map_values(
             " columns : %s",
             source_column)
     for old, new in mapping:
+
+        new = "'%s'"% new
+
+        if old == True:
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade.py]: You must use `if old is True`

File: openupgradelib/openupgrade.py [hunk 2]
```diff
@@ -734,20 +735,35 @@ def map_values(
...
+            old = 'NOT NULL'
+            op = 'IS'
+        elif old == False:
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade.py]: Use `elif old is False:`

File: openupgradelib/openupgrade.py [hunk 3]
```diff
@@ -734,20 +735,35 @@ def map_values(
...
+            old = 'NULL'
+            op = 'IS'
+        else:
+            old = "'%s'"% old
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade.py]: put an space before `%`

Comment (kurkop) [inline openupgradelib/openupgrade.py]: @pedrobaeza before which %?

Comment (pedrobaeza) [inline openupgradelib/openupgrade.py]: `old = "'%s'" % old` (for avoiding lint errors)

File: openupgradelib/openupgrade.py [hunk 4]
```diff
@@ -734,20 +735,35 @@ def map_values(
             " columns : %s",
             source_column)
     for old, new in mapping:
+
+        new = "'%s'"% new
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade.py]: I forgot to highlight the same problem here:
```
openupgradelib/openupgrade.py:739:21: E225 missing whitespace around operator
```

Comment (pedrobaeza) [inline openupgradelib/openupgrade.py]: And please make only 1 commit for cleanliness

Comment (kurkop) [inline openupgradelib/openupgrade.py]: Ok, I made the changes.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by StefanRijnhart** [comment]
Please rewrite the commit's message, which now includes Git's hints on rebasing. Next time, it is fine to leave the commits as they are instead of rebasing your proposal into a single commit after review.

**Comment by pedrobaeza** [comment]
I was the one who ask him to rebase to only 1 commit, because we put a lot of commits for simple patches that makes the commit history very awful. Putting correctly the commit message is enough for ensuring the correctness of the full PR.

**Comment by StefanRijnhart** [comment]
Alright, a matter of taste I guess.

**Comment by pedrobaeza** [comment]
@StefanRijnhart, if code is OK for you, I can rewrite commit message when merging

**Comment by StefanRijnhart** [comment]
@pedrobaeza Deal! :+1:

**Pull request merged**